components/src/dynamo/frontend/sglang_prepost.py

```
@@ -147,6 +147,62 @@ def _random_call_id() -> str:
     return random_call_id()
 
 
+def _decode_single_token(tokenizer, token_id: int) -> str | None:
+    try:
+        return tokenizer.decode([token_id], skip_special_tokens=False)
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)